### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by thiago-buarqque**

### CODE REVIEW

File: modules/apps/static/portal-osgi-web/portal-osgi-web-wab-generator-impl/src/main/java/com/liferay/portal/osgi/web/wab/generator/internal/processor/WabProcessor.java [hunk 1]
```diff
@@ -287,6 +287,15 @@ private File _convertToClientExtensionBundleDir(
 			frontendPathString += "/";
 		}
 
+		boolean languageDetected = false;
+
+		String languagePathString = pluginPackageProperties.getProperty(
+			_LIFERAY_CLIENT_EXTENSION_BATCH, "content/");
```
Comment (rotty3000) [inline modules/apps/static/portal-osgi-web/portal-osgi-web-wab-generator-impl/src/main/java/com/liferay/portal/osgi/web/wab/generator/internal/processor/WabProcessor.java:293]: this should be a new attribute name (change `BATCH` for `LANGUAGE`)

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java [hunk 2]
```diff
@@ -859,6 +863,9 @@ private String _validateAndGetClassificationGrouping(
 		else if (_groupFrontend.containsAll(classifications)) {
 			return "frontend";
 		}
+		else if (_groupLanguage.containsAll(classifications)) {
```
Comment (rotty3000) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java:866]: remove this change, you don't add new classifications.

File: modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/internal/client/extension/client-extension.properties [hunk 3]
```diff
@@ -1,4 +1,5 @@
 batch.classification=batch
+language.classification=language
```
Comment (rotty3000) [inline modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/internal/client/extension/client-extension.properties:2]: change this to `language.classification=batch`

File: modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/language/Dockerfile.tpl [hunk 4]
```diff
@@ -0,0 +1 @@
+FROM liferay/noop
```
Comment (rotty3000) [inline modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/language/Dockerfile.tpl:1]: remove this whole file

File: modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/language/LCP.json.tpl [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (rotty3000) [inline modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/language/LCP.json.tpl:1]: remove this whole file

File: workspaces/liferay-sample-workspace/gradle.properties [hunk 6]
```diff
@@ -9,4 +9,5 @@ liferay.workspace.node.package.manager=yarn
 liferay.workspace.product=dxp-2023.q4.5
 liferay.workspace.themes.dir=themes
 liferay.workspace.wars.dir=modules
+liferay.workspace.home.dir=/home/me/dev/bundles/master
```
Comment (rotty3000) [inline workspaces/liferay-sample-workspace/gradle.properties:12]: you can delete this line and create a `gradle-local.properties` which will do the same but be ignored by git

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java [hunk 7]
```diff
@@ -907,6 +914,8 @@ private void _writeToOutputFile(
 		"configuration");
 	private static final Set<String> _groupFrontend = Sets.newHashSet(
 		"configuration", "frontend");
+	private static final Set<String> _groupLanguage = Sets.newHashSet(
+		"configuration", "language");
```
Comment (rotty3000) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java:918]: remove this line

**Review by rotty3000 — CHANGES_REQUESTED** [review]

File: workspaces/liferay-sample-workspace/gradle.properties (hunk 6 above)
Comment (thiago-buarqque) [reply workspaces/liferay-sample-workspace/gradle.properties:12]: Oh didn't know that. I wouldn't send it for real though.

Comment (rotty3000) [reply workspaces/liferay-sample-workspace/gradle.properties:12]: Oh I know... I was just letting you in on this little secret workflow improvement :)

Comment (thiago-buarqque) [reply workspaces/liferay-sample-workspace/gradle.properties:12]: Now I am part of a secret society :wink:

**Pull request closed without merging**